certipy/test/test_certipy.py

```diff
@@ -373,10 +374,19 @@ def test_certs():
         ) as server:
             # Execute/Verify
             url = f"https://{server.host}:{server.port}"
-
             # Fails without specifying a CA for verification
-            with pytest.raises(SSLError):
-                requests.get(url)
+            with pytest.raises(URLError, match="SSL"):
+                with urlopen(url):
+                    pass
+
+            ssl_context = ssl.create_default_context(ssl.Purpose.SERVER_AUTH, cafile=ca_record["files"]["cert"])
+            ssl_context.verify_mode = ssl.CERT_REQUIRED
+            ssl_context.load_default_certs()
+            ssl_context.load_cert_chain(ca_record["files"]["cert"], ca_record["files"]["key"])
+            # currently required to pass on 3.13
+            # if hasattr(ssl, "VERIFY_X509_STRICT"):
+            #     ssl_context.verify_flags &= ~ssl.VERIFY_X509_STRICT
```

Comment:
VERIFY_X509_STRICT is in the default flags as of Python 3, which checks AuthorityKeyIdentity